src/mongo/db/storage/wiredtiger/wiredtiger_record_store_oplog_stones.h

```diff
@@ -129,12 +136,8 @@ class WiredTigerRecordStore::OplogStones {
     // database, and false otherwise.
     bool _isDead = false;
 
-    // Maximum number of stones to keep in the deque before the background reclaim thread should
-    // truncate the oldest ones. Does not include the stone currently being filled. This value
-    // should not be changed after initialization.
-    size_t _numStonesToKeep;
     // Minimum number of bytes the stone being filled should contain before it gets added to the
-    // deque of oplog stones. This value should not be changed after initialization.
```

Comment:
I'm still unclear what the repercussions are of changing this value to the overall logic. Can you give me an argument for why this is OK despite the original warning?